tests/tenant_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build !slim
```

Comment:
By looking at this, I'm thinking taking the !slim (embedded) as the default. Then we don't need to add these build tags.

slim -> embedPg

Reply:
I agree that we don't need to specify !slim since that's the default. It corresponds to our docker image where the non-slim one uses bytebase/bytebase

But I still prefer to keep the word "slim" instead of using "embedPg", because we may embed/not embed other binaries (e.g. mail server).

@d-bytebase @tisonkun thoughts?

Reply:
Yeah, I don't like embedPg either. How about "rich" or "nonSlim"?

Reply:
I would go for omni or mono

Reply:
omni is the one used by GitLab, their all-in-one binary called omnibus

Reply:
omni SG

Reply:
Updated.